stable_baselines3/common/off_policy_algorithm.py

```diff
@@ -427,14 +428,14 @@ def _dump_logs(self) -> None:
         """
         Write log.
         """
-        time_elapsed = time.time() - self.start_time
-        fps = int((self.num_timesteps - self._num_timesteps_at_start) / (time_elapsed + 1e-8))
+        time_elapsed_ns = max(time.time_ns() - self.start_time, sys.float_info.epsilon)
```

Comment:
why not multiply here directly by 1e9? (as you do it twice later)

Reply:
Good point, I think I was trying to avoid rounding error, but it's fine to divide by `1e9` after computing the delta.